src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +141,39 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
     }
 
+    /**
+     * Adds metadata to request packet to enable telemetry and tracking. Request packet will be
+     * extended with following parameters:
+     * {
+     *   "meta": {
+     *     "sdk": {
+     *       "version": "v1.2.1",
+     *       "lang": "java",
+     *       "lang_version": "oracle-v1.9.0",
+     *       "platform": "linux",
+     *       "platform_version": "10"
+     *     }
+     *   }
+     * }
+     *
+     * @param mixed    $requestPacket
+     *
+     * @return mixed
+     */
+    private function addMeta($requestPacket) {
+        $requestPacket['meta'] = [
+            'sdk' => [
+                'version' => 'hardcoded for now',
+                'lang' => 'php',
+                'lang_version' => 'hmm',
```

Comment:
What should be the lang version here? `5.6`? I found this in readme.md:
> This SDK should run on PHP 5.6 and is compatible with PHP 7.

Reply:
We need to extract the actual running PHP version. Most likely from this: https://secure.php.net/manual/en/function.phpversion.php

```php
php > echo phpversion();
7.2.7
```

Reply:
👍